latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+* A `default` EKS Node class as created when the `general-purpose` or `system` Managed Node Pools are enabled.
+
+== Step 1: Deploy a GPU workload
+
+In this example, you will create a NodePool for NVIDIA inference based workloads that requires 45GB GPU memory. With EKS Auto Mode, you use Kubernetes scheduling constraints to define your instance requirements. 
+
+To deploy the Amazon EKS Auto Mode `NodePool` and the sample `workload`, review the following NodePool and Pod definition and save as `nodepool-gpu.yaml` and `pod.yaml`:
+
+*nodepool-gpu.yaml*
+
+[source,yaml]
+----
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: gpu
+spec:
+  disruption:
+    budgets:
+    - nodes: 10%
+    consolidateAfter: 1h
+    consolidationPolicy: WhenEmpty
+  template:
+    metadata: {}
+    spec:
+      nodeClassRef:
+        group: eks.amazonaws.com
+        kind: NodeClass
+        name: default
+      requirements:
+        - key: "karpenter.sh/capacity-type"
+          operator: In
+          values: ["on-demand"]
+        - key: "kubernetes.io/arch"
+          operator: In
+          values: ["amd64"]
+        - key: "eks.amazonaws.com/instance-family"
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+*pod.yaml*
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidia-smi
+spec:
+  nodeSelector:
+    eks.amazonaws.com/instance-gpu-memory: "45776"
+    eks.amazonaws.com/compute-type: auto
+  restartPolicy: OnFailure
+  containers:
+  - name: nvidia-smi
+    image: nvidia/cuda:12.8.0-cudnn-devel-ubuntu20.04
+    args:
+    - "nvidia-smi"
+    resources:
+      requests:
+        memory: "30Gi"
+        cpu: "3500m"
+        nvidia.com/gpu: 1
+      limits:
+        memory: "30Gi"
+        nvidia.com/gpu: 1
+----
+
+Note the `eks.amazonaws.com/compute-type: auto` selector requires the workload be deployed on an Amazon EKS Auto Mode node.
+
+Apply the NodePool and workload to your cluster.
+
+[source,bash]
+----
+kubectl apply -f nodepool-gpu.yaml
+kubectl apply -f pod.yaml
+----
+
+You should see the following output:
+
+[source,bash]
+----
+nodepool.karpenter.sh/gpu configured created
+pod/nvidia-smi created
+----
+
+Wait about one minute, and check the nodes in your cluster. You should now see a new node provisioned in your Amazon EKS Auto Mode cluster:
+
+[source,bash]
+----
+> kubectl get nodes
+
+NAME        TYPE          CAPACITY    ZONE         NODE                  READY   AGE
+gpu-dnknr   g6e.2xlarge   on-demand   us-west-2b   i-02315c7d7643cdee6   True    76s
+----
+
+== Step 2: Validate
+
+You can see Amazon EKS Auto Mode launched a `g6e.2xlarge` rather than an `g6.2xlarge` as the workload required an instance with at least 45GB of GPU memory, according to the following Kubernetes scheduling constraints:
+
+[source,yaml]
+----
+...
+  nodeSelector:
+    eks.amazonaws.com/instance-gpu-memory: "45776"
```

Comment:
Suggested to replace with: `eks.amazonaws.com/instance-gpu-name: l40s` as per above.